src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -429,6 +436,14 @@ private void BuildApplicationInitializerBody(IndentedStringBuilder writer, XamlO
 			}
 		}
 
+        private void InitializeRemoteControlClient(IndentedStringBuilder writer)
+        {
+            if (_isDebug)
+            {
+                writer.AppendLineInvariant($"global::Uno.UI.RemoteControl.RemoteControlClient.Initialize(GetType());");
```

Comment:
is there any way to opt out of Hot Reload, eg if it happens to cause problems in some context?

Reply:
There is not, but this is interesting. Will adjust, thanks.